src/components/TodoContainer/TodoContainer.jsx

```diff
@@ -128,13 +131,33 @@ const TodoContainer = () => {
 
     let completedTodos = todoList.filter(todo => todo.completed);
 
+    const sortTodos = (value) => {
+        switch (value) {
+            case "titleAsc":
+                setTodoList(prevTodoList => [...prevTodoList].sort((a, b) => a.title < b.title ? -1 : 1));
```

Comment:
Instead of explicitly defining this case, let `titleAsc` go to the default case to minimize duplication. Add a comment in that default section to specify the behavior